lib/degica_datadog/config.rb

```diff
@@ -30,7 +30,7 @@ def service
       end
 
       def version
-        @version || ENV.fetch("_GIT_REVISION", "unknown")
+        @version || (ENV.fetch("PLATFORM", "") + ENV.fetch("_GIT_REVISION", "unknown"))
```

Comment:
should we add `-` in between?

Reply:
I had the same thought, but to make it easy, I was about to define the env var as `PLATFORM=cpp-`.
On the second thought, we should implement this properly. We should add `-` only when `PLATFORM` is provided.

Reply:
ah then lets do ternary with string interpolation? if platform exist render it with - in the end?

Reply:
Updated at [8e33e86](https://github.com/degica/degica_datadog/pull/8/commits/8e33e86fc0e2252f28573c9cc55eaa8f38e179ac).

Reply:
@Daivasmara Could you kindly review the code again, just in case?